sos/report/plugins/kea.py

```diff
@@ -0,0 +1,50 @@
+# Copyright (C) 2024 Red Hat, Inc., Jose Castillo <jcastillo@redhat.com>
+
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.report.plugins import (Plugin, RedHatPlugin, UbuntuPlugin)
+
+
+class Kea(Plugin, UbuntuPlugin):
+    """
+    Kea is a Kea is the next generation of DHCP software, developed by Internet
+    Systems Consortium (ISC). It supports both the DHCPv4 and DHCPv6 protocols
+    along with their extensions, e.g. prefix delegation and dynamic updates to
+    DNS.
+    """
+    short_desc = 'Kea DHCP and DDNS server from ISC'
+
+    plugin_name = "kea"
+    packages = ("kea", )
+    services = ('kea-ctrl-agent', 'kea-dhcp-ddns-server',
+                'kea-dhcp4-server', 'kea-dhcp6-server',)
+
+    def setup(self):
+        self.add_copy_spec([
+            "/etc/kea/*",
+        ])
+
+    def postproc(self):
+        """ format is "password": "kea", """
+        self.do_path_regex_sub(
+            '/etc/kea/*',
+            r'(^\s*"password":\s*)(".*"),',
+            r'\1********'
+        )
+
+
+class RedHatKea(Kea, RedHatPlugin):
+
+    def setup(self):
+        super().setup()
+        self.add_cmd_output([
+            "keactrl status",
+        ])
```

Comment:
This looks like it exists for Ubuntu - https://manpages.ubuntu.com/manpages/kinetic/man8/keactrl.8.html

A quick Google makes it look like it's part of the standard packaging, so we can make this a regular collection and shift this plugin to `IndependentPlugin`. Worst case scenario on a system that doesn't have it, we get a log message about the command not being found.

Reply:
OK, I've reworked it a bit so it covers Ubuntu better, thanks for finding this out.